recipes/slamdunk/meta.yaml

```diff
@@ -40,9 +39,10 @@ requirements:
     - ncurses
     - zlib
     - bzip2
-    - curl
+    - curl >7.59
     - xz
     - libdeflate
+    - openssl=1.0
```

Comment:
Is this really needed? Or should some dependencies be rebuilt to handle current openssl version?N

Reply:
I will fix this in the next release, but until then this is the quickfix to keep a viable version out.